angular-ngrx-scss/src/app/pull-requests/components/pull-requests-list/pull-requests-list.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
cleanup: We can delete this test file. In general, ones that are only ensuring the component creates we don't need to keep. :)